Sources/SwerverCore/Router/Routes.swift

```diff
@@ -22,7 +22,7 @@ public class Routes {
     }
     
     public func fetchAllActions(url: URL) -> [RequestMethod:HttpAction] {
-        return allRoutes[url]!
+        return allRoutes[url] ?? [:]
```

Comment:
This is a cool use of `??`. What exactly does `??` do?

Reply:
?? is used to provide a fallback if the predecessor is nil, in this case, it'll return an empty dictionary

Reply:
Very cool.